src/functionaltests/java/com/ericsson/ei/notifications/ttl/TestTTLSteps.java

```diff
@@ -47,13 +47,13 @@
         "aggregated.collection.ttlValue:1",
         "notification.failAttempt:1",
         "spring.data.mongodb.database: TestTTLSteps",
-        "failed.notification.database-name: TestTTLSteps-failedNotifications",
+        "failed.notification.collection-name: TestTTLSteps-failedNotifications",
         "rabbitmq.exchange.name: TestTTLSteps-exchange",
         "rabbitmq.consumerName: TestTTLStepsConsumer"})
 public class TestTTLSteps extends FunctionalTestBase {
     private static final Logger LOGGER = LoggerFactory.getLogger(TestTTLSteps.class);
     private static final String BASE_URL = "localhost";
-    private static final String ENDPOINT = "/missed_notification";
+    private static final String ENDPOINT = "/notification-endpoint";
```

Comment:
notification-endpoint? If this test does not fail on wrong endpoint investigate why it does not fail.

Reply:
It's just an arbitrary notification endpoint. It's the notification meta that is mocked for the test.

Reply:
Changed it to a less confusing name.